DockService.App/Properties/launchSettings.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
File should not be existant.

Use the following the remove ignored files: [How to make Git “forget” about a file that was tracked but is now in .gitignore?
Ask](https://stackoverflow.com/a/19095988/3563975)